components/email/src/Email.svelte

```diff
@@ -387,7 +387,11 @@
   }
 
   async function handleThread(event: MouseEvent | KeyboardEvent) {
-    if (activeThread.messages.length <= 0) {
+    const messageType = isThreadADraftEmail(activeThread)
```

Comment:
Since we only have 2 message types for now, I'd prefer to just use a boolean to track this. Something like `isDraftEmail`. In the future, if we have more message types we can create an enum to track them.